prac_05/hex_colours.py

```diff
@@ -0,0 +1,13 @@
+COLOUR_TO_CODE = {"Absolute Zero": "#0048ba", "Acid Green": "#b0bf1a", "AliceBlue": "#f0f8ff",
+                  "Alizarin crimson": "#e32636",
+                  "Amaranth": "#e52b50", "Amber": "#ffbf00", "Amethyst": "#9966cc",
+                  "AntiqueWhite": "#faebd7", "AntiqueWhite1": "#ffefdb", "AntiqueWhite2": "#eedfcc"}
+for colour_name, code in COLOUR_TO_CODE.items():
+    print(f"{colour_name} is {code}")
+colour_name = input("Enter colour: ").lower()
+if colour_name.lower() in (name.lower() for name in COLOUR_TO_CODE):
+    for name, code in COLOUR_TO_CODE.items():
+        if colour_name == name.lower():
+            print(name, "is", code)
+else:
+    print("Invalid colour")
```

Comment:
Code looks solid. You may want to add a blank line or two between the loops and statements (marked) just for easier readability (see further comments for suggested space locations).

Looks good otherwise :+1:

Reply:
thank you 😊